cwl_tes/main.py

```diff
@@ -123,13 +132,18 @@ def main(args=None):
 
     if parsed_args.token:
         try:
+            header = jwt.get_unverified_header(parsed_args.token)
```

Comment:
That is a good idea; the sequence is now to first check the command line parameter (which gets a default value from an environment variable), and if nothing is there, it will try to load a file with the name of the key. If that fails, an error is raised.